settings.gradle.kts

```diff
@@ -1 +1,6 @@
-rootProject.name = "sigstore-java"
+rootProject.name = "sigstore-java-root"
```

Comment:
I've no a good name for "the root project".

`sigstore-java` is taken by the java library itself, so it would probably be confusing to have `sigstore-java` under `sigstore-java` project. That is why I renamed to root to `sigstore-java-root`.

An alternative option would be just `sigstore`, however, it sounds too generic for me.

Reply:
I don't care too much about this. It will never be exposed to users, only devs.